src/Helper/ObjectParser.php

```diff
@@ -136,20 +294,20 @@ public function getAttributeTranslator(): ?AttributeTranslatorInterface
     }
 
     /**
-     * @return array<string, ReflectionProperty>
+     * Returns Reflection properties parsed from {@see $source} in accordance with {@see $propertyVisibility} and
+     * {@see $skipStaticProperties} values. Repetitive calls utilize cache if it's enabled in {@see $useCache}.
+     *
+     * @return array<string, ReflectionProperty> A mapping between Reflection property names and their values.
      */
-    public function getReflectionProperties(): array
+    private function getReflectionProperties(): array
```

Comment:
Keep this method as public. It's need for `FormCollector` implement in `yiisoft/form`.